prime-router/src/main/kotlin/cli/tests/CompareData.kt

```diff
@@ -322,7 +324,7 @@ warnings: ${warnings.joinToString()}
     fun compare(
         expected: File,
         actual: File,
-        format: Report.Format,
+        format: Report.Format?,
```

Comment:
What is the use case where we do not know the format? Would we be comparing things where we do not have a format?